tycho-versions-plugin/src/main/java/org/eclipse/tycho/versions/pom/PomFile.java

```diff
@@ -245,4 +276,25 @@ private String getElementValue(String name) {
     public boolean isMutable() {
         return isMutable;
     }
+
+    private static boolean isCiFriendly(String v) {
+        return v != null && v.contains("${");
+    }
+
+    private void resolveCiFriendlyVersion() {
```

Comment:
can this be made to return the resolved version instead of manipulate the resolved version directly?